setup.py

```diff
@@ -14,7 +14,7 @@
     install_requires=[
         'jupyterhub>=0.8',
         'pyYAML',
-        'kubernetes==6.*',
+        'kubernetes==7.*',
```

Comment:
Let's also take this opportunity to switch this to `>=`. Pinning with `==` in library dependencies is frowned upon in Python.